test/main.js

```diff
@@ -179,6 +179,22 @@ test('ky.extend()', async t => {
 	await server.close();
 });
 
+test('ky.extend() throws when given non-object argument', t => {
+	[
+		true,
+		666,
+		'hello',
+		[],
+		null,
+		() => {},
+		Symbol('ky')
+	].forEach(value => {
```

Comment:
Make the list of types a separate variable and use a `for-of` loop instead. (Just my preference. Makes it more readable)